### ISSUE TITLE
Solar activity not found when starting in monthly predictions

### ISSUE THREAD

**Issue by gaffarelj** [tudat-team/tudat#38]
It appears that no solar activity can be found when a simulation starts more than 7257.5 JD after J2000 (the 14th of November 2019). This was discovered using the `nrlmsise00` atmosphere model that uses solar activity as an input.

However, starting before 7257.5 JD and propagating for years works.

Coincidentally, the [space weather resource file](https://github.com/tudat-team/tudat-resources/blob/0952fd3f3d2a8350bb18f27a6a32351b45082336/resource/space_weather/sw19571001.txt) switches from daily predictions to monthly predictions the14th of November 2019.

Coincidence? No, manually updating the data to [a more recent space weather](http://celestrak.com/SpaceData/SW-All.txt), the switch from daily predictions to monthly predictions is on the 20th of February 2022. Sure enough, the same error as before still happens, but now for this new date.

**In short**: starting a simulation at a time where the space weather is in _monthly_ predictions causes the solar activity data to be inaccessible while starting in observations or _daily_ predictions and then later switching to _monthly_ predictions seems to work fine.

**Comment by DominicDirkx** [comment]
Good point, I think we should update this on two fronts:
* Also load the monthly data from the file, and use this for interpolation if needed
* Automatically download the latest space weather file when it becomes available. This may be best done in conjunction with an automatic downloader tha @ggarrett13 has been working on.

**Comment by DominicDirkx** [comment]
I've made a few modifications to the NRLMSISE functions:

https://github.com/tudat-team/tudat/tree/feature/nrlmsise_update
https://github.com/tudat-team/tudatpy/tree/feature/nrlmsise_update

It's now possible to use the predictions from the space weather file. As it turns out, this was quite straightforward, as these data were already loaded.

I have extended the unit tests to check if it's possible to use the monthly predictions (it is), but I don't have any reference data directly handy. Doing this would require a separate manual run of the NRLMSISE00 code.

**Comment by DominicDirkx** [comment]
Closing issues; remaining issue(s) being tracked through https://github.com/tudat-team/tudat/issues/40
